src/test/java/org/openmrs/module/mirebalais/smoke/helper/SmokeTestDriver.java

```diff
@@ -24,6 +24,7 @@ public SmokeTestDriver() {
         // these options allow us to run chrome as root (which we really shouldn't do) see https://bbs.archlinux.org/viewtopic.php?id=196353
         ChromeOptions options = new ChromeOptions();
         options.addArguments(Arrays.asList("no-sandbox","start-fullscreen"));
+        options.setHeadless(true);
```

Comment:
Actually: although we don't need headless when we run as part of CI, we definitely need the ability to run the tests locally and watch the output in order to debug... does this still work, and, if so, can we make this a configurable parameter?

Reply:
OK, I've made this configurable via environment variable like the other settings @mogoodrich

Reply:
Thanks! Let's just make sure we confirm it runs properly locally with headless=false.